### PR TITLE
Part 2: Basic console app

### PR DESCRIPTION
**Pull request by benjimarshall**

### CODE REVIEW

File: Bookish.DataAccess/BookishDb.cs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+using System.Configuration;
+using System.Data;
+using System.Data.SqlClient;
+using Bookish.DataAccess.Records;
+using Dapper;
+
+
+namespace Bookish.DataAccess
+{
+    public class BookishDb
```
Comment (FridaTveit) [inline Bookish.DataAccess/BookishDb.cs:12]: We usually call classes like this which interact with the db a service, e.g. `BookishService` 🙂

File: Bookish.DataAccess/BookishDb.cs [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    {
+        private readonly IDbConnection db;
```
Comment (FridaTveit) [inline Bookish.DataAccess/BookishDb.cs:14]: `connection` is probably more descriptive? And avoids abbreviations 🙂

File: Bookish.DataAccess/BookishDb.cs [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+        public BookishDb()
+        {
+            db = new SqlConnection(@"Server = localhost; Database = Bookish; Trusted_Connection = True;");
+        }
+
+        public IEnumerable<Book> GetBooks()
+        {
+            var sqlString = "SELECT TOP 100 [isbn],[title],[authors] FROM [books]";
```
Comment (FridaTveit) [inline Bookish.DataAccess/BookishDb.cs:23]: I don't think you need the square brackets. And this won't necessarily return all the books 🙂

File: Bookish.DataAccess/BookishDb.cs [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+            return (List<Book>)db.Query<Book>(sqlString);
```
Comment (FridaTveit) [inline Bookish.DataAccess/BookishDb.cs:25]: I don't think you need the cast 🙂

File: Bookish.DataAccess/BookishDb.cs [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+namespace Bookish.DataAccess
+{
+    public class BookishDb
+    {
+        private readonly IDbConnection db;
+
+        public BookishDb()
+        {
+            db = new SqlConnection(@"Server = localhost; Database = Bookish; Trusted_Connection = True;");
```
Comment (FridaTveit) [inline Bookish.DataAccess/BookishDb.cs:18]: You shouldn't need the `@` 🙂

File: Bookish.DataAccess/BookishDb.cs [hunk 6]
```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Data;
+using System.Data.SqlClient;
+using Bookish.DataAccess.Records;
+using Dapper;
+
+
```
Comment (FridaTveit) [inline Bookish.DataAccess/BookishDb.cs:9]: Unneeded empty line 🙂

**Review by FridaTveit — COMMENTED** [review]
A few minor suggestions, otherwise looks good! 🙂

File: Bookish.DataAccess/BookishDb.cs (hunk 6 above)
Comment (benjimarshall) [reply Bookish.DataAccess/BookishDb.cs]: Removed in 74b24a3 🙂

File: Bookish.DataAccess/BookishDb.cs (hunk 5 above)
Comment (benjimarshall) [reply Bookish.DataAccess/BookishDb.cs]: Removed in 74b24a3 🙂

File: Bookish.DataAccess/BookishDb.cs (hunk 4 above)
Comment (benjimarshall) [reply Bookish.DataAccess/BookishDb.cs]: Removed in 74b24a3 🙂

File: Bookish.DataAccess/BookishDb.cs (hunk 3 above)
Comment (benjimarshall) [reply Bookish.DataAccess/BookishDb.cs]: Removed and fixed in 74b24a3 🙂

File: Bookish.DataAccess/BookishDb.cs (hunk 2 above)
Comment (benjimarshall) [reply Bookish.DataAccess/BookishDb.cs]: Yep, changed in 74b24a3 🙂

File: Bookish.DataAccess/BookishDb.cs (hunk 1 above)
Comment (benjimarshall) [reply Bookish.DataAccess/BookishDb.cs]: That's hand to know! Changed in 74b24a3 🙂

File: Bookish.ConsoleApp/Program.cs [hunk 7]
```diff
@@ -7,7 +7,7 @@ class Program
     {
         static void Main(string[] args)
         {
-            var bookishDb = new BookishDb();
+            var bookishDb = new BookishService();
```
Comment (FridaTveit) [inline Bookish.ConsoleApp/Program.cs:10]: Very minor but this variable should probably be renamed to be consistent with the name of the class 🙂

**Review by FridaTveit — COMMENTED** [review]
One tiny thing otherwise looks great 🙂

File: Bookish.ConsoleApp/Program.cs (hunk 7 above)
Comment (benjimarshall) [reply Bookish.ConsoleApp/Program.cs]: Yep, fixed in fb73ee2 🙂

**Review by FridaTveit — APPROVED** [review]
Looks great 🙂

**Pull request merged**